README.md

```diff
@@ -56,36 +58,82 @@ import { ReactSearch } from "@vectara/react-search";
 />;
 ```
 
-### Configuration options
+#### <u>Configuration options</u>
 
-#### `customerId` (required)
+##### `customerId` (required)
 
 Every Vectara account is associated with a customer ID. You can find your customer ID by logging into the [Vectara Console](https://console.vectara.com/) and opening your account dropdown in the top-right corner.
 
-#### `corpusId` (required)
+##### `corpusId` (required)
 
 After you [create a corpus](https://docs.vectara.com/docs/console-ui/creating-a-corpus), you can find its ID by navigating to the corpus and looking in the top-left corner, next to the corpus name.
 
-#### `apiKey` (required)
+##### `apiKey` (required)
 
 API keys enable applications to access data inside of corpora. Learn how to [create a **QueryService** API key](https://docs.vectara.com/docs/console-ui/manage-api-access#create-an-api-key).
 
-#### `apiUrl` (optional)
+##### `apiUrl` (optional)
 
 By default, React-Search sends query requests to the Vectara servers. If you want to use a proxy server, you can configure this option with the URL of your proxy.
 
-#### `placeholder` (optional)
+##### `placeholder` (optional)
 
 Configure the placeholder text in the search modal's input.
 
-#### `isDeeplinkable` (optional)
+##### `isDeeplinkable` (optional)
 
 Defaults to `false`. Set this option if you want to persist a search query to a URL parameter. This will enable users to share or bookmark the URL. Loading the URL will automatically open the search modal and search for the query that's stored in the URL.
 
-#### `openResultsInNewTab` (optional)
+##### `openResultsInNewTab` (optional)
 
 Defaults to `false`. Set this option if you want a search result to open in a new tab.
 
+### Use your own views with the useSearch hook
```

Comment:
I'm not sure I understand what "use your own views" means. How about rephrasing this as "Power your own search UI with the useSearch hook"?